tool/teleport/common/teleport.go

```diff
@@ -83,6 +92,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	status := app.Command("status", "Print the status of the current SSH session.")
 	dump := app.Command("configure", "Generate a simple config file to get started.")
 	ver := app.Command("version", "Print the version of your teleport binary.")
+	join := app.Command("join", "Download the Teleport CA and SSH host keys and update the SSH config to use them")
```

Comment:
Have we already settled on `join` as the name here? The description here doesn't sound like it would be for something like `join`.

I would either pick a better subcommand name or change the help message (maybe something like `Join an SSH server to a Teleport cluster`).

Do we foresee any cases when we might want to do an agentless join for other protocols? If so, we might want a better subcommand for that purpose too. Maybe `teleport join openssh`?

Reply:
`teleport join openssh` sounds find to me.